### ISSUE TITLE
Filter examples in documentation don't match

### ISSUE THREAD

**Issue by Jeltje** [nextflow-io/nextflow#1908]
## Bug report 

https://www.nextflow.io/docs/latest/operator.html, under Filter, under the Tip at the very end:

```
In the above example the filter condition is wrapped in curly brackets, instead of round brackets, since it 
specifies a closure as the operator’s argument. This just is a language syntax-sugar for 

filter({ it.toString().size() == 1 })
```

However, this is the example it references:
```
Channel
    .from( 1, 2, 3, 4, 5 )
    .filter { it % 2 == 1 }
    .view()
```
These filters are not the same (the one under Tip outputs all 5 numbers).

**Comment by Jeltje** [comment]
I promise this is the last one, at least for a while :-)
This manual is so well written, it's a joy to read. I want to help make it just a bit better.

**Comment by Jeltje** [comment]
Oh, to be extra nitpicky: `This just is` should probably be `This is just`

**Comment by pditommaso** [comment]
Thanks! Solved by 376d13bb
